src/functions/template-tags/general.php

```diff
@@ -841,6 +841,17 @@ function tribe_get_minimum_cost() {
 
 		$costs = array_map( 'tribe_map_cost_array_callback', $costs );
 		foreach ( $costs as $index => $value ) {
+			// try to find the lowest numerical value in a possible range
+			if ( preg_match( '/^(-?[\d]+)[^\d\.]+([\d\.]+)/', $value, $matches ) ) {
```

Comment:
It would be cool to have that Regular expression on a method that returns an `Array`.